diagnostic.md

```diff
@@ -35,19 +35,19 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
+ <!-- Answer Starts Here --> This is an absolute path because it references the root directory
 
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
+ <!-- Answer Starts Here --> We would use a relative path, because URL's are always relative.
```

Comment:
It would be an absolute because a url, needs the root (which domain to target and the path within the directory. URL's are always absolute.